lib/prefix.c

```diff
@@ -1395,3 +1466,26 @@ unsigned prefix_hash_key(void *pp)
 		     offsetof(struct prefix, u.prefix) + PSIZE(copy.prefixlen),
 		     0x55aa5a5a);
 }
+
+unsigned int prefix_opaque_hash_key(void *pp)
```

Comment:
prefix_hash_key?  Again why do we need a separate function for this?

Reply:
done